EyeSeeTea/d2-cloner
# Added all fields instead of id,name and users in the userGroup query …

…after update with put method

### :pushpin: References
Issue: http://who-dev.essi.upc.edu:8083/issues/3525

### :tophat: What is the goal?

Avoid overwrite usergroupaccesses when removing users from the usergroup


### :memo: How is it being implemented?

Replacing the fields from id,name,users to * to replace the usergroup by the usergroup with all its attributes.

### :boom: How can it be tested?
- wait training clone in weekend
- run clone with removeFromGroups action and check the usergroup permissions.
